Source/WebKit/WebProcess/GPU/graphics/RemoteImageBufferProxy.h

```diff
@@ -254,6 +260,7 @@ class RemoteImageBufferProxy : public WebCore::ConcreteImageBuffer<BackendType>
         ASSERT(resolutionScale() == 1);
         auto& mutableThis = const_cast<RemoteImageBufferProxy&>(*this);
         mutableThis.flushDrawingContextAsync();
+        prepareForBackingStoreChange();
```

Comment:
How can we guarantee `hasPendingFlush()` will return false in this case when it is called from `prepareForBackingStoreChange()`?

Reply:
True. I changed this to use the new old `backingStoreWillChange()`  (same as it was in the initial patch)